### PR TITLE
add packages, add test data

### PR DESCRIPTION
**Pull request by s-kganz**
## Overview

- Added boto, boto3, ftplib, netCDF4, networkx, IPython, ipywidgets, traitlets to environment.yaml
- Added testwind.csv, testcurrent.csv to dev_notebooks/keenan

Connects #8

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -10,4 +10,11 @@ dependencies:
   - obspy
   - requests
   - lxml
-  - boto3
+  - boto
```
Comment (lsetiawan) [inline environment.yml]: This package is actually called `boto3` not `boto` even tough it ends up `import boto`. See: https://anaconda.org/conda-forge/boto3

File: environment.yml [hunk 2]
```diff
@@ -10,4 +10,11 @@ dependencies:
...
+  - ftplib
```
Comment (lsetiawan) [inline environment.yml]: `ftplib` is part of the python standard library, no need to be listed here. See: https://docs.python.org/3/library/ftplib.html

File: environment.yml [hunk 3]
```diff
@@ -10,4 +10,11 @@ dependencies:
...
+  - netCDF4
```
Comment (lsetiawan) [inline environment.yml]: The `xarray` package up above brings in `netCDF4` so no need to be listed here. Highest level packages is all that's needed.

File: environment.yml [hunk 4]
```diff
@@ -10,4 +10,11 @@ dependencies:
...
+  - networkx
+  - IPython
```
Comment (lsetiawan) [inline environment.yml]: `IPython` should already be a part of the jupyter notebook, no need to list here.

File: environment.yml [hunk 5]
```diff
@@ -10,4 +10,11 @@ dependencies:
...
+  - ipywidgets
+  - traitlets
+  - boto3
```
Comment (lsetiawan) [inline environment.yml]: Already listed up above. See my comment: https://github.com/cormorack/notebooks/pull/10/files#r187813315

**Comment by s-kganz** [comment]
requested changes added to environment.yml @lsetiawan

**Comment by lsetiawan** [comment]
Looks like now you’re having a conflict with the base branch. So please rebase to the latest commit before I can merge this. Thanks!

**Comment by lsetiawan** [comment]
Conflict has been fixed. Merging.

**Pull request merged**